en_US/durability/management.md

```diff
@@ -86,11 +86,11 @@ By default, the `epoch_bits` parameter is configured to 20 (~1 s), striking a ba
 
 ## CLI Commands
 
-The following CLI commands are available for managing the durable storage:
+The following CLI commands are available for managing the durable sessions:
```

Comment:
Please revert. These commands don't manage the sessions; they manage the storages.